.github/renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "config:best-practices",
+    ":gitSignOff",
+    ":rebaseStalePrs",
+    "group:allNonMajor",
+    "group:linters",
+    "group:test",
+    ":preserveSemverRanges",
+    ":pinOnlyDevDependencies"
+  ],
+  "labels": ["kind/dependency upgrade"],
+  "npm": {
+    "minimumReleaseAge": "1 day"
+  },
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "groupName": "GitHub Actions"
+    },
+    {
+      "matchPackagePatterns": ["^@backstage/"],
```

Comment:
This one's interesting. We generally tell adopters to deliberately upgrade their actual Backstage repos using the cli tool. We still don't have a great story for plugin makers to be able to guarantee that "oh my plugin is built against and works with Backstage versions 1.19-1.23" (with whatever sub package versions that that entails).

For now, no action. Let's try it the way it's proposed here.

Reply:
FYI You are already grouping this as you extend [`best-practices`](https://docs.renovatebot.com/presets-config/#configbest-practices) https://docs.renovatebot.com/presets-monorepo/#monorepobackstage
Only the group name will change.